HelpSource/Reference/randomSeed.schelp

```diff
@@ -52,6 +55,32 @@ thisThread.randSeed = 1923;
 Array.fill(7, r.value);
 ::
 
+When you set code::randSeed:: on a Routine, it switches to its own random number generator, independent of any other. Subsequently, routines created within the independent-RNG routine will inherit the separate random number generator.
```

Comment:
change first "it" to "Routine" for clarity. second sentence is a bit awkward, consider "Any new Routines created within the context of this Routine will inherit its unique RNG."